### PR TITLE
APPDEV-1144 Integrate build with Slack

### PR DESCRIPTION
**Pull request by Bert-R**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -38,17 +36,19 @@ pipeline {
...
         archiveArtifacts 'Yona/BuildOutput/**/*.ipa'
       }
       post {
         always {
           junit 'Yona/BuildOutput/**/*.xml'
-          
         }
-        
+        success {
+          slackSend color: 'good', channel: '#dev', message: "iOS app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} succeeded"
```
Comment (vishal-revdiwala) [inline Jenkinsfile]: I want to know difference between `${BUILD_NUMBER}` and `${env.BUILD_NUMBER}`.

File: Jenkinsfile [hunk 2]
```diff
@@ -38,17 +36,19 @@ pipeline {
...
+        }
+        failure {
+          slackSend color: 'bad', channel: '#dev', message: "iOS app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} failed"
```
Comment (vishal-revdiwala) [inline Jenkinsfile]: I want to know difference between `${BUILD_NUMBER}` and `${env.BUILD_NUMBER}`.

File: Jenkinsfile (hunk 1 above)
Comment (Bert-R) [reply Jenkinsfile]: The first one refers to a Groovy variable named ``BUILD_NUMBER``, the second one to a map entry ``BUILD_NUMBER`` in the Groovy variable ``env``, which holds the environment variables.

File: Jenkinsfile (hunk 2 above)
Comment (Bert-R) [reply Jenkinsfile]: See above

File: Jenkinsfile (hunk 1 above)
Comment (vishal-revdiwala) [reply Jenkinsfile]: Ok Thanks.

**Pull request merged**